Userland/Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -131,7 +131,7 @@ static ViewBoxTransform scale_and_align_viewbox_content(SVG::PreserveAspectRatio
 
 void SVGFormattingContext::run(Box const& box, LayoutMode layout_mode, AvailableSpace const& available_space)
 {
-    // FIXME: This a bunch of this thing is an ad-hoc hack.
+    // FIXME: Some of this is ad-hoc (hacks).
```

Comment:
Having had another look, I can't. There used to be the quite big hacks here, but they have since been replaced with more spec-compliant stuff. There are some other some specific FIXMEs in the `SVGFormattingContext` now, and I think those should be more useful :^)